Add EffectsPlus plugin

scottgeliza opened the pull request:
EffectsPlus is a set of commands aiming to help improve immersion for cutscenes in hosted roleplay.
- Made by @melismau(tic)

Features:
- TweenBlind: gradual :blind command, slowly darkens the player's screen within the given time frame
- TweenPitch: gradual :pitch command, increases/decreases pitch to the given value within the provided time frame
- TweenVolume: gradual :volume command, slowly increases/decreases volume within the given time frame
- ScreenImage improvements: supports transparency and color3, and now also occupies the entire screen
-> TweenImage: a command used to gradually change transparency and/or color3 values of existing ScreenImage instances
- Typewriter text (with RichText support; just pending ROBLOX filter implementation.........) a la visual novel style

- ...And also some slight changes to the :unblind command so as to support removing :tweenblind and :screenimage

scottgeliza committed:
Add files via upload

EffectsPlus is a set of commands aiming to help improve immersion for cutscenes in hosted roleplay.
- Made by @melismau(tic)

Features:
- TweenBlind: gradual :blind command, slowly darkens the player's screen within the given time frame
- TweenPitch: gradual :pitch command, increases/decreases pitch to the given value within the provided time frame
- TweenVolume: gradual :volume command, slowly increases/decreases volume within the given time frame
- ScreenImage improvements: supports transparency and color3, and now also occupies the entire screen
-> TweenImage: a command used to gradually change transparency and/or color3 values of existing ScreenImage instances
- Typewriter text (with RichText support; pending ROBLOX filter implementation.........) a la visual novel style

scottgeliza committed:
Changed to RBXMX

scottgeliza committed:
Delete Server/Server-EffectsPlus.rbxm

silkyrie commented:
Updated!
(I just now learned about rbxmx files...)

scottgeliza committed:
Delete Server/Server-EffectsPlus.rbxmx

Updating to new version

scottgeliza committed:
Add files via upload

Updated EffectsPlus
-- Text string filtering now functions properly
-- Verified command functionality (everything should work as intended now...)
-- New commands: Player-Specific ZaWarudo and Custom Roblox Notifications

silkyrie commented:
Everything is now up-to-date! (I think...)
Could I get an update on the pull request?

scottgeliza committed:
Update Server-EffectsPlus.rbxmx

Commented out a debug line; print(currentChar)
hopefully shouldn't clog up devconsoles anymore
